x-pack/plugins/index_management/public/application/sections/home/index_list/details_page/details_page_overview/aliases_details.tsx

```diff
@@ -58,7 +59,7 @@ export const AliasesDetails: FunctionComponent<{ aliases: Index['aliases'] }> =
               <EuiFlexItem grow={false}>
                 <EuiText
                   css={css`
-                    font-size: ${euiThemeVars.euiFontSizeL};
+                    font-size: ${euiTheme.size.l};
```

Comment:
@mattkime for font-sizes, there's a separate scale: `euiTheme.font.scale.*`, so this would be:

```suggestion
                    font-size: ${euiTheme.font.scale.l};
```

Same in all the font-size cases below 👇🏻

Reply:
thank you!

Reply:
@mattkime mea culpa, and this is already merged... 👀 

But it's not `euiTheme.font.scale.l`, this is without the unit :( We need to use `euiFontSize(themeContext, 'l')`.

Reply:
Reverting - https://github.com/elastic/kibana/pull/204400